package.json

```diff
@@ -37,15 +37,15 @@
     "@types/command-line-args": "^5.0.0",
     "@types/get-stdin": "^5.0.1",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^11.12.0",
+    "@types/node": "^11.13.20",
     "@types/nunjucks": "^3.1.0",
     "@types/rimraf": "^2.0.2",
     "assert-rejects": "^1.0.0",
-    "c8": "^5.0.1",
+    "c8": "^5.0.4",
     "codecov": "^3.0.4",
     "espower-typescript": "^9.0.0",
     "google-gax": "^1.6.0",
-    "gts": "^1.0.0",
+    "gts": "^0.9.0",
```

Comment:
This change should not be here (and make sure you do `npx gts fix` when you change back to `gts` v1.0.0).